### PR TITLE
[Feature] Add About Project Section

### PR DESCRIPTION
**Pull request by KobbyMmo**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # ([179117193](https://www.pivotaltracker.com/story/show/179117193))

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
image

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/AboutProject/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  const classes = useStyles(props);
+
+  return (
+    <div className={classes.root}>
+      <div className={classes.background}>
+        <Image objectFit="cover" src={AboutProjectBg} layout="fill" />
+      </div>
+      <Section classes={{ root: classes.section }}>
+        <Grid container>
+          <Grid item xs={12} md={9} lg={6} className={classes.textContainer}>
```
Comment (kilemensi) [inline src/components/AboutProject/index.js:21]:
```suggestion
          <Grid item xs={12} md={9} lg={6} className={classes.content}>
```

File: src/components/AboutProject/useStyles.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  },
+  section: {
+    zIndex: 1,
+    height: "100%",
+  },
+  button: {
+    textDecoration: "none",
+    "&:hover": {
+      textDecoration: "none",
+    },
```
Comment (kilemensi) [inline src/components/AboutProject/useStyles.js:31]: Aren't these on theme already? If not, that's where they should be. _Unless_ you're saying this behaviour is unique to this component.

**Review by kilemensi — COMMENTED** [review]
👍🏾

**Comment by kilemensi** [comment]
And for a millionth time, where is the CMS integration?

File: src/components/AboutProject/index.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import Section from "@/pesayetu/components/Section";
+
+function AboutProject({ subtitle, title, href, ctaText, ...props }) {
+  const classes = useStyles(props);
+
+  return (
+    <div className={classes.root}>
+      <div className={classes.background}>
+        <Image objectFit="cover" src={AboutProjectBg} layout="fill" />
+      </div>
```
[the remark this answers is not in the document]
Comment (KobbyMmo) [reply src/components/AboutProject/index.js:18]: When we were having the problem with the background Images, @KhadijaMahanga suggested this so we also get optimization from next @kilemensi

[the remark this answers is not in the document]
Comment (KhadijaMahanga) [reply src/components/AboutProject/index.js]: @KobbyMmo missed this mention! Yeah, that seemed to be the solution most people agreed with on this discussion https://github.com/vercel/next.js/discussions/18357#discussioncomment-132523.

But I guess they don't have hundreds on background images as we do 😆

**Review by KhadijaMahanga — COMMENTED** [review]
@KobbyMmo Looks good 🚀 But something is down `lg`

Screenshot from 2021-08-20 11-57-11

File: src/components/AboutProject/index.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+          <Grid item xs={12} md={9} lg={6} className={classes.content}>
+            <Typography variant="h4" className={classes.title}>
+              {title}
+            </Typography>
+            <Typography variant="body1" className={classes.subtitle}>
+              {subtitle}
+            </Typography>
+            <Button component={Link} href={href} variant="text">
+              {ctaText}
+            </Button>
```
[the remark this answers is not in the document]
Comment (KhadijaMahanga) [reply src/components/AboutProject/index.js:28]: @KobbyMmo ignore this. I didn't notice the Button with variant text

**Review by KhadijaMahanga — APPROVED** [review]
🚀

**Review by kilemensi — APPROVED** [review]
🚀

**Review by gertie-sheshe — APPROVED** [review]
🎉

**Pull request merged**